proto/v1/sql_service.proto

```diff
@@ -242,18 +242,16 @@ message PrettyResponse {
 message CheckRequest {
   string statement = 1;
 
-  Engine engine = 2;
-
   enum ChangeDatabaseType {
     CHANGE_DATABASE_TYPE_UNSPECIFIED = 0;
     DDL = 1;
     DML = 2;
     SDL = 3;
   }
-  ChangeDatabaseType change_database_type = 3;
+  ChangeDatabaseType change_database_type = 2;
 
   // Format: instances/{instance}/databases/{databaseName}
-  string database = 4;
+  string database = 3;
```

Comment:
Maybe not changing the number but marking "unused 2"

Reply:
This RPC is not used yet.